test/test_any_automated_install.py

```diff
@@ -679,7 +679,7 @@ def test_FTL_detect_aarch64_no_errors(host):
     mock_command(
         'ldd',
         {
-            '/bin/ls': (
+            '/bin/bash': (
```

Comment:
```suggestion
            *: (
```
I don't remember offhand what the proper way to say "Return this no matter what is called"

Reply:
Let's leave this as it is and mock `which` to return `/bin/sh` for `which sh`.